package.json

```diff
@@ -38,7 +38,7 @@
     "chalk": "^1.1.3",
     "ember-cli-babel": "^7.1.2",
     "ember-cli-import-polyfill": "^0.2.0",
-    "ember-get-config": "",
+    "ember-get-config": "^0.5.0",
```

Comment:
Can you set this to `^1.0.0`?

Reply:
It should be backwards compatible. I think anything that can't build with 1.0 should be considered a bug in that repo or a bug in get-config. Pinning this library to something else will cause more problems than it solves.

Reply:
Understand 👍 I re-checked in fresh ember app and I see `ember-cli-mirage` have a problem with this error, not `ember-pikaday`.

I can change version to `^1.0.0` or close this PR. Just let me know

Reply:
I'd love to merge with `^1.0.0`. I think that's better than the current blank and probably stable for a very long time (hopefully longer than this addon needs to exist).

Reply:
Sure, version was changed, thanks